### DISCUSSION TITLE
Problem with YOURLS - Cannot open /admin/index.php

### DISCUSSION THREAD

**Discussion by hernanpopper** [YOURLS/YOURLS#3562] [Q&A]
I thought it was a good idea to upgrade to 1.9.2, and I'm not sure what I broke, but I get an error 500 when I try to access /admin/index.php

It works and redirects, and I can access /admin/upgrade.php, but the admin index page always fails. 

I'm desperate as I use this for my small business and losing it would mean losing hundreds of shortened links sent via emails in the last year or so.

Can anyone offer some advise?

Thanks so much!
![sVFMyPHEvy](https://user-images.githubusercontent.com/104868017/235182142-96c29990-56ed-496b-bb62-4bfadd411942.png)
![dQ1r98roaX](https://user-images.githubusercontent.com/104868017/235182146-80017cd1-bce3-4ee9-8270-9cab80bc218e.png)

**Comment by LeoColomb** [comment]
Checkout https://yourls.org/docs/guide/troubleshooting/first-steps
